AssertionError  in data_quality.ipynb

When I tried to apply data_quality.ipynb I got from github today
 to the data in 2021-present as follows:
```
files = glob.glob("/fefs/aswg/data/real/OSA/DL1DataCheck_LongTerm/night_wise/all/DL1_datacheck_202[1-4]*.h5") # since 2021-
```
the assertion error occurred like this: 
```
' # Check that now the runsummary entries match the runs in runlist (in the same order)'
assert(np.allclose(runlist, runsummary['runnumber']))

---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
Cell In[20], line 2
      1 # Check that now the runsummary entries match the runs in runlist (in the same order)
----> 2 assert(np.allclose(runlist, runsummary['runnumber']))

AssertionError: 

```
When I tried with
```
files = glob.glob("/fefs/aswg/data/real/OSA/DL1DataCheck_LongTerm/night_wise/all/DL1_datacheck_202[4]*.h5") 
```
or 
```
files = glob.glob("/fefs/aswg/data/real/OSA/DL1DataCheck_LongTerm/night_wise/all/DL1_datacheck_202[1-3]*.h5") 
```
, they worked well.


At the present, there are two faulty runs like:
```
Removing run 8091 from runsummary table!
Removing run 17855 from runsummary table!,
```
But run 17855 does not seem to be removed from runsummary. Since the lengths of runlist and runsummary table are the same, the different run may be removed.

![image](https://github.com/user-attachments/assets/eb75608c-a1b7-4b7d-a09f-15bce0c5cd02)

--- a/notebooks/data_quality.ipynb
+++ b/notebooks/data_quality.ipynb
@@ -125,7 +125,7 @@
     "# files = glob.glob(\"/fefs/aswg/data/real/OSA/DL1DataCheck_LongTerm/v0.10/2023????/DL1_datacheck_2023*.h5\")\n",
     "# \n",
     "# Example: load all the 2023 files processed (with any version - the latest available for each night): \n",
-    "files = glob.glob(\"/fefs/aswg/data/real/OSA/DL1DataCheck_LongTerm/night_wise/all/DL1_datacheck_2023*.h5\n",
+    "files = glob.glob(\"/fefs/aswg/data/real/OSA/DL1DataCheck_LongTerm/night_wise/all/DL1_datacheck_202[1-4]*.h5\")\n",
     "\n",
     "files.sort()"
    ]
@@ -501,8 +501,8 @@
     "    if r in runlist:\n",
     "        continue\n",
     "    print('Removing run', r, 'from runsummary table!')\n",
-    "    \n",
-    "    runsummary = runsummary.drop(np.where(runsummary['runnumber']==r)[0])"
+    "    runsummary.drop(np.where(runsummary['runnumber']==r)[0], inplace=True)\n",
+    "    runsummary.reset_index(inplace=True, drop=True)"
    ]
   },
   {
@@ -1820,6 +1820,14 @@
    "metadata": {},
    "outputs": [],
    "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0101cc95",
+   "metadata": {},
+   "outputs": [],
+   "source": []
   }
  ],
  "metadata": {
@@ -1838,7 +1846,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.4"
+   "version": "3.11.8"
   }
  },
  "nbformat": 4,

